### PR TITLE
Fix README example

### PR DESCRIPTION
**Pull request by gitbuda**

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Review by BradReesWork — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -320,8 +320,12 @@ Please see https://github.com/rapidsai/cugraph/releases/tag/v21.10.00a for the l
 - PR #1174 Fix bugs in MNMG pattern accelerators and pattern accelerator based implementations of MNMG PageRank, BFS, and SSSP
 - PR #1233 Temporarily disabling C++ tests for 0.16
 - PR #1240 Require `ucx-proc=*=gpu`
+<<<<<<< HEAD
+
+=======
 - PR #1241 Fix a bug in personalized PageRank with the new graph primitives API.
 - PR #1249 Fix upload script syntax
+>>>>>>> origin/branch-0.17
```
Comment (rlratzel) [inline CHANGELOG.md:328]: Looks like some markers from a merge conflict were left in and still need to be removed.

**Review by rlratzel — CHANGES_REQUESTED** [review]
Thanks for fixing the README. I noticed a minor problem with the CHANGELOG that needs to be fixed, but otherwise this looks good.

**Review by rlratzel — APPROVED** [review]

**Comment by BradReesWork** [comment]
rerun tests

**Comment by ajschmidt8** [comment]
ok to test

**Pull request merged**